storage_manager_new.py

```diff
@@ -5,46 +5,50 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.load_items_from_file()
 
     def load_items_from_file(self):
-        Item.instances = []
+        self.items = []
```

Comment:
Do you intentionally clear previously instantiated `self.items` here? If so, it might be better to just use python list `clear()` method. Also, we might want to make it clear in method name that we are not only loading items from file here, but we also erase any previous existing items from there.

Reply:
Yes, I clear `self.items` intentionally, to avoid multiplying instances when this method is called as a part of delete_item method.
I will work on method name and use `clear()` method as well, thanks!